python/lsst/sims/maf/batches/srdBatch.py

```diff
@@ -95,14 +92,14 @@ def astrometryBatch(colmap=None, runName='opsim',
 
     subgroup = metadata
 
-    raCol = colmap['ra']
-    decCol = colmap['dec']
-    degrees = colmap['raDecDeg']
+    raCol, decCol, degrees, ditherStacker, ditherMeta = radecCols(ditherStacker, colmap, ditherkwargs)
+    # Don't want dither info in subgroup (too long), but do want it in bundle name.
+    metadata = combineMetadata(metadata, ditherMeta)
```

Comment:
I think the `combineMetadata` might not be working as intended for the `run_srd.py` script. When using the following command: 
~~~
run_srd.py baseline2018a/data/astro-lsst-01_2022.db --runName baseline2018a --outDir baseline2018a/srd_dither  --ditherStacker RandomDitherPerNightStacker --sqlConstraint 'night <= 720'
~~~

The resulting metric results files have names where the dither stacker used is repeated twice. For example: `baseline2018a_fO_night_lt_720_RandomDitherPerNight_RandomDitherPerNight_HEAL.npz`

I used the same `--ditherStacker` and `--sqlConstraint` flags with the `run_all.py` and `run_nvisits.py` and there the dither stacker used is not repeated in the file names.

From `run_nvisits.py` for example: `baseline2018a_NVisits_WFD_night_lt_720_RandomDitherPerNight_all_bands_HEAL.npz`